app/src/main/java/org/wikipedia/edits/EditsContributionsFragment.kt

```diff
@@ -47,7 +47,7 @@ import java.util.concurrent.TimeUnit
 import kotlin.collections.ArrayList
 import kotlin.collections.HashMap
 
-class SuggestedEditsContributionsFragment : Fragment(), SuggestedEditsContributionsHeaderView.Callback {
+class EditsContributionsFragment : Fragment(), EditsContributionsHeaderView.Callback {
```

Comment:
Not everything needs to have the `Edits` prefix, and in fact having `Edits...` in front of all the names makes it sound weird.  This could easily be `ContributionsFragment`. You get the idea.